cloudygames/models.py

```diff
@@ -8,15 +18,68 @@ class Game(models.Model):
     address = models.CharField(max_length=45)
     users = models.ManyToManyField(User)
 
+class GameSession(models.Model):
+    player = models.ForeignKey(User)
+    game = models.ForeignKey(Game)
+    controller = models.IntegerField()
+
+    def joinGame(self, _game):
```

Comment:
Model plays an important role in general. Regardless whether one view need
it or not. Model should serve for more general purpose instead of a
particular view. After join game function being called, I am expecting a
session back.

GameSessionFullException is just a custom exception. you will throw it in
this function if it reached the maximum player.
Then you will handle it in the caller.

On Sat, Feb 6, 2016 at 7:29 PM, gabriellaMT <notifications@github.com>
wrote:

> In cloudygames/models.py
> <https://github.com/insert-coin/cloudyweb/pull/4#discussion_r52101034>:
>
> > @@ -8,15 +18,68 @@ class Game(models.Model):
> >      address = models.CharField(max_length=45)
> >      users = models.ManyToManyField(User)
> >
> > +class GameSession(models.Model):
> > +    player = models.ForeignKey(User)
> > +    game = models.ForeignKey(Game)
> > +    controller = models.IntegerField()
> > +
> > +    def joinGame(self, _game):
>
> As the creation of the GameSession is inside the view, I believe join_game
> would not need the return value of a game session. I apologize if this is
> incorrect though.
>
> Can you explain more about GameSessionFullException?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/insert-coin/cloudyweb/pull/4/files#r52101034>.
>